spec/forms/candidate_interface/degree_wizard_spec.rb

```diff
@@ -765,24 +765,6 @@
       end
     end
 
-    describe '#sanitize_country' do
-      let(:stored_data) { {}.to_json }
-      let(:attrs) { { uk_or_non_uk: 'uk', country: 'FR', current_step: :country } }
-
-      it 'clears country' do
-        wizard = described_class.new(store, attrs)
-        expect(wizard.country).to be_nil
-        expect(wizard.uk_or_non_uk).to eq 'uk'
-      end
```

Comment:
Just want to try and find the historic context as to why we were sanitising this in the first place!